where-for-dinner/README.md

```diff
@@ -52,3 +52,13 @@ Note that the event channel protocols are not specified as they can be swapped o
 The default build and deployment architecture uses Spring Cloud Streams as the asynchronous messaging implementation with RabbitMQ as the default binding.  The messaging topology is depicted in the following abridged architecture diagram: 
 
 ![](doc/images/SCSMessaging.png)
+
+### KNative Eventing 
+
+The *Cloud Events* option of the Where For Dinner application contains build configuration that removes Spring Cloud Streams dependencies and relies on the Spring Cloud Function webflux library for receiving events over HTTP.  Knative eventing fully embraces this paradigm and integrates seamlessly with functions exposed via the Spring Cloud Functions module.
+
+The architecture still uses Spring Cloud Streams as the messaging implementation for initially emitting searches, and a KNative `source` is inserted to create a bridge between the Spring Cloud Streams binding and the downstream Knative eventing concepts.  For the RabbitMQ implementation, a KNative RabbitMQ Source is declared to read events from a RabbitMQ exchange and propagate them through Knative eventing down to the subscribing services.  The following is a diagram of the Knative event concepts used for Where For Dinner:
+
+![](doc/images/KNativeEventing.png)  
```

Comment:
Rename `KNativeEventing.png` to `KnativeEventing.png`

Reply:
Updated.  Reference to updated as well in documentation.